code/modules/mob/living/carbon/human/life.dm

```diff
@@ -440,24 +440,31 @@
 		var/failed_inhale = 0
 		var/failed_exhale = 0
 
-		if(species.has_organ["breathing apparatus"] && src.get_species() == "Vaurca")
-			var/obj/item/organ/vaurca/breathingapparatus/L = internal_organs_by_name["breathing apparatus"]
-			if(isnull(L))
-				poison_type = null
-			else if(L.is_broken())
-				poison_type = "oxygen" //if Vaurca breathing apparatus breaks, oxygen becomes poisonous.
+		if(species.has_organ["filtration bit"] && src.get_species() == "Vaurca")
+			var/obj/item/organ/vaurca/filtrationbit/F = internal_organs_by_name["filtration bit"]
+			if(isnull(F))
+				poison_type = "oxygen" //if Vaurca does not have filter, oxygen becomes poisonous
+
+			else if(F.is_broken())
+				poison_type = "oxygen" //if Vaurca filter breaks, oxygen becomes poisonous.
+
+			else if(!isnull(F) && !F.is_broken())

```

Comment:
Change this to a simple `else`.

There's no need to run both checks again and see if they return negative. If neither one of the above has returned positive already, then they won't here. `else` is thus implied, and should be used.